udata/core/dataset/models.py

```diff
@@ -276,14 +271,12 @@ def __str__(self):
     }
 
     meta = {
-        'allow_inheritance': True,
         'indexes': [
             '-created_at',
             'slug',
-            'organization',
             'resources.id',
             'resources.urlhash',
-        ],
+        ] + db.Owned.meta['indexes'],
```

Comment:
But where does it come from ?
The `meta['indexes']` the only way (I know) to get the mixin default indexes.
It relies on mongoengine default (documented) behavior.